pkg/http/client_test.go

```diff
@@ -0,0 +1,74 @@
+package http
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"net/http"
+	"net/http/httptest"
+	"strings"
+	"testing"
+
+	"github.com/dcos/dcos-cli/pkg/config"
+	"github.com/stretchr/testify/require"
+)
+
+func TestGet(t *testing.T) {
+	query := `{"cluster":"DC/OS"}`
+	ts := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		w.Write([]byte(query))
+	}))
+	defer ts.Close()
+
+	conf, err := config.FromString(`
+[core]
+
+dcos_url = "` + ts.URL + `"
+`)
+	require.NoError(t, err)
+
+	client := New(conf)
+
+	resp, err := client.Get("/")
```

Comment:
Similarly to what is done for TestPost, we should make assertions in the http test server handler to make sure the client sends the expected request (a `GET` request to the `/` path).